app.py

```diff
@@ -13,28 +13,30 @@
 external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']
 
 # Dash App
-app = dash.Dash(__name__, title='GHG Cloud', external_stylesheets=external_stylesheets, prevent_initial_callbacks=True)
+app = dash.Dash(__name__, title='GHG Cloud', external_stylesheets=external_stylesheets)
 server = app.server
 
 # Connect to InfluxDB
 client = InfluxDBClient.from_config_file("influx_config.ini")
 query_api = client.query_api()
 
-def get_influxdb_data():
+def get_influxdb_data(duration):
     ## Query data as pandas dataframe
-    df = query_api.query_data_frame('from(bucket:"co2") '
-                                    '|> range(start: -1h) '
-                                    '|> filter(fn: (r) => r.host == "6cb1b8e43a19bdb3950a118a36af3452")'
-                                    '|> pivot(rowKey:["_time"], columnKey: ["_field"], valueColumn: "_value") '
+    df = query_api.query_data_frame('from(bucket:"co2")'
+                                    f'|> range(start: -{duration})'
+                                    '|> filter(fn: (r) => r.host == "af1ae06960bff131b214d73d7747d3b5")'
```

Comment:
Not sure why the sensor ID just changed...

Reply:
What do you mean @brianstrauch ? As in why did you change it or did the query stop working?

Reply:
Looks like there was a problem with the other sensor last night where it went offline briefly, so that's why your query stopped working. We should fix the website so that it's robust to this and only displays sensors that have data in #1